packages/react-button/src/components/Button/Button.scss

```diff
@@ -249,8 +149,8 @@
   font-size: var(--button-iconSize, inherit);
   font-weight: normal;
 
-  &._disabled {
-    color: var(--button-disabled-iconColor, var(--button-disabled-contentColor));
+  ._disabled & {
```

Comment:
Why are we doing this change?

Reply:
it was wrong previously and am fixing it to be correct here;
icon slot doesn't have `_disabled` modifier class name. only root does